packages/base/Breadcrumbs/src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,21 +1,40 @@
 import type { ReactNode } from 'react'
 import React, { forwardRef } from 'react'
 import type { BaseProps } from '@toptal/picasso-shared'
-import { Breadcrumbs as MuiBreadcrumbs } from '@material-ui/core'
-import { ChevronRight16 } from '@toptal/picasso-icons'
-import './styles'
+import { Typography } from '@toptal/picasso-typography'
+
+import insertSeparator from './utils/insertSeparator'
 
 export interface Props extends BaseProps {
   /** Content of Breadcrumbs */
   children: ReactNode
 }
 
-export const Breadcrumbs = forwardRef<HTMLElement, Props>(function Breadcrumbs(
-  props,
-  ref
-) {
-  return <MuiBreadcrumbs ref={ref} separator={<ChevronRight16 />} {...props} />
-})
+export const Breadcrumbs = forwardRef<HTMLDivElement, Props>(
+  function Breadcrumbs(props, ref) {
+    const { children } = props
+
+    const allItems = React.Children.toArray(children).map((child, index) => (
+      // here is fine to use index as key (https://github.com/jsx-eslint/eslint-plugin-react/issues/1123)
+      // eslint-disable-next-line react/no-array-index-key
+      <li className='p-0 m-0' key={`child-${index}`}>
+        {child}
+      </li>
+    ))
```

Comment:
We shouldn't create a new key for children elements, as this can fail in unexpected ways; keys should be preserved as the same as the child element. For that, React provides the [`Children.map` function](https://react.dev/reference/react/Children#children-map) which automatically merge the keys from the children.

```suggestion
    const allItems = React.Children.map(children, (child, index) => (
      <li className='p-0 m-0'>
        {child}
      </li>
    ))
```

Reply:
Acknowledge that! Thx!